scripts/ci/all.sh

```diff
@@ -4,6 +4,6 @@ srcdir=`dirname $0`
 test -z "$srcdir" && srcdir=.
 . ${srcdir}/common.sh
 
-DEFAULT_COMPOSE_ARGS="master --no-apm-server-dashboards --no-apm-server-self-instrument --with-agent-rumjs --with-agent-go-net-http --with-agent-nodejs-express --with-agent-python-django --with-agent-python-flask --with-agent-ruby-rails --with-agent-java-spring --force-build --build-parallel"
+DEFAULT_COMPOSE_ARGS="${ELASTIC_STACK_VERSION:-'master'} ${BUILD_OPTS} --no-apm-server-self-instrument --no-apm-server-dashboards --with-agent-rumjs --with-agent-go-net-http --with-agent-nodejs-express --with-agent-python-django --with-agent-python-flask --with-agent-ruby-rails --with-agent-java-spring --force-build"
```

Comment:
i'd prefer the options stay sorted unless there is a good reason, so `--no-apm-server-dashboards` before `--no-apm-server-self-instrument`

Reply:
done